modules/services/ogc-api-records/service/src/main/java/org/fao/geonet/ogcapi/records/MvcConfigurer.java

```diff
@@ -28,14 +29,13 @@
   public LocaleResolver localeResolver() {
     final AcceptHeaderLocaleResolver resolver = new AcceptHeaderLocaleResolver();
 
-    List<Language> languageList = languageRepository.findAll();
-    List<Locale> supportedLocales = new ArrayList<>();
-    languageList.forEach(lang -> {
-      Locale locale = Locale.forLanguageTag(lang.getId());
-      if (locale != null) {
-        supportedLocales.add(locale);
-      }
-    });
+    List<Locale> supportedLocales = languageRepository
+        .findAll() //
```

Comment:
what is the purpose of `//` ?

Reply:
there are sometimes tools performing auto indent, so to forbid them inline all ?

Reply:
In this case configure the tool. .editorconfig is here for that no?